src/tests/unittests/native/GraphMockTests.cpp

```diff
@@ -0,0 +1,50 @@
+// Copyright 2021 The WebNN-native Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+
+#include "mocks/GraphMock.h"
+#include "webnn_native/NamedInputs.h"
+#include "webnn_native/NamedOutputs.h"
+
+namespace webnn_native { namespace {
+
+    using ::testing::Test;
+
+    class GraphMockTests : public Test {
+      protected:
+        GraphMock graphMock;
+    };
+
+    TEST_F(GraphMockTests, AddOps) {
+        EXPECT_CALL(graphMock, AddConstant).Times(1);
+        EXPECT_TRUE(graphMock.AddConstant(nullptr).IsSuccess());
```

Comment:
The test case show that it can success even through we pass a `nullptr`, does it make sense?

Reply:
If we don't specify an implementation for the mock method, it will run with its default action which can get nullptr.